authentication/certificate-types-descriptions.adoc

```diff
@@ -63,6 +63,44 @@ to serve as a placeholder until you configure a custom default certificate. Do
 not use Operator-generated default certificates in production clusters.
 ====
 
+[discrete]
+== Workflow
+
+.Custom certificate workflow
+
+image::custom-workflow.png[custom ingress certificate workflow]
+
+
+.Default certificate workflow
+
+image::default-workflow-4-5.png[default ingress certificate workflow]
+
+image:darkcircle-0.png[20,20] An empty `defaultCertificate` field causes the Ingress Operator to use its self-signed CA to generate a serving certificate for the specified domain.
+
+image:darkcircle-1.png[20,20] The default CA certificate and key generated by the Ingress Operator. Used to sign Operator-generated default serving certificates.
+
+image:darkcircle-2.png[20,20] The wildcard default serving certificate, created by the Ingress Operator and signed using the generated default CA certificate.
```

Comment:
`wildcard default serving certificate` is applicable for the default workflow. For the custom workflow, this is the user-provided cert.